### PR TITLE
fix: hide outline when ephemeral update is in progress

### PR DESCRIPTION
**Pull request by kof**
Ref https://github.com/webstudio-is/webstudio/issues/3914
Closes https://github.com/webstudio-is/webstudio/issues/3800

## Description
This allows to see borders/outlines when changing ephimeral styles e.g. scrubbing border-outline width or color

This is not 100% fixing the problem, as you still can't see the border/outline when you change non ephimeral value, but there is not much we can actually do, because outline has to be blue for clarity, making it look like the user color, would be potentially very confusing.

## Steps for reproduction

1. change any ephimeral styles like padding drag/drop or scrub any input
2. see outline is hidden during that change

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env` file

### CODE REVIEW

**Comment by johnsicili** [comment]
Click into border color field and click away. No border appears again until refresh

https://github.com/user-attachments/assets/2be3d636-be18-41eb-a51e-8bb4869738a2

**Comment by johnsicili** [comment]
It seems to only disappear on scrubbing. But when I click in and arrow, it doesn't. This is probably needed because when you're fine-tuning to the point where you don't want to see the border, you need fine control, which isn't obtained easily via scrubbing.

https://github.com/user-attachments/assets/4dbf9ca9-93b0-4f18-84fb-efd9fcf217a0

**Comment by kof** [comment]
@johnsicili same in wf, I also don't see a good way to solve this, because outline happens outisde and you can't generally let it get covered by something else. The only thing  I could imagine is some button or shortcut that would hide the outline until you turn it on again

**Comment by kof** [comment]
@johnsicili I can add a view menu point "Show canvas outlines" and add a shortcut for it

File: apps/builder/app/builder/features/style-panel/shared/use-style-data.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ import {
 import { useStyleInfo } from "./style-info";
 import { serverSyncStore } from "~/shared/sync";
 import { $ephemeralStyles } from "~/canvas/stores";
+import { $isEphemeralUpdateInProgress } from "~/builder/shared/nano-states";
```
Comment (TrySound) [inline apps/builder/app/builder/features/style-panel/shared/use-style-data.ts:21]: would checking $ephemeralStyles work for you?

**Comment by johnsicili** [comment]
The outline doesn't always come back. Click the position dropdown, and click out. No outline

File: apps/builder/app/builder/features/style-panel/shared/use-style-data.ts (hunk 1 above)
Comment (kof) [reply apps/builder/app/builder/features/style-panel/shared/use-style-data.ts:21]: actually it might

**Review by TrySound — APPROVED** [review]

**Pull request merged**